travis/Dockerfile.armhf

```diff
@@ -0,0 +1,21 @@
+FROM multiarch/ubuntu-core:armhf-xenial
+
+ENV ARCH=armhf
+
+SHELL ["bash", "-x", "-c"]
+
+RUN apt-get update && \
+    apt-get install -y gcc g++ wget make gnupg zip git subversion libglib2.0-dev automake libtool patch zlib1g-dev libcairo2-dev libssl-dev libfuse-dev fuse libgtest-dev vim-common desktop-file-utils zsync bzip2 xz-utils libcurl4-gnutls-dev libjpeg-dev libpng-dev cmake && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/NixOS/patchelf.git && \
```

Comment:
This should _not_ be in the done in the Dockerfile. It must be run during every build, as it used to be. _Only this_ guarantees that the script that creates the final AppImage contains patchelf 0.8, and not some broken system-provided version.